examples/niscope_acquire_waveform/NIScopeAcquireWaveform.seq

```diff
@@ -4588,16 +4588,16 @@
 																					<value>pin_names</value>
 																				</Name>
 																				<ArgumentValue classname='ExprValue'>
-																					<value>"Pin1,Pin2,Pin3,Pin4"</value>
+																					<value>{"Pin1","Pin2","Pin3","Pin4"}</value>
```

Comment:
Did this have the wrong data type before?

Reply:
Yes Brad.  Adding the screenshot of previous version

Initial load of seq file:

![image](https://user-images.githubusercontent.com/13461073/235476686-e523aa9e-62e3-4f05-9e80-db5af3c269f1.png)


After Reloading measurement params:

![image](https://user-images.githubusercontent.com/13461073/235476833-420e1ded-ee47-49ab-9a6d-ebc47d0679d5.png)